Create NASA Grant Blogpost (#610)
open-life-science/open-life-science.github.io: _posts/2023-05-05-NASA-grant-announcement.md

---
layout: post

> title: "One small step for NASA, one big step for open research: OLS and MetaDocencia to deliver OpenCore Science Training"

Reviewer: Should we split the title into a title and a subtitle?

authors:
- yochannah
---

> For many, 2023 is the [Year of Open Science](https://www.whitehouse.gov/ostp/news-updates/2023/01/11/fact-sheet-biden-harris-administration-announces-new-actions-to-advance-open-and-equitable-research/). 
> 
> OLS and MetaDocencia (MD) are delighted to announce that we will be participating in the Year of Open Science by delivering Open Science training for NASA, using materials originally developed by [a huge group of open researchers worldwide](https://opensciency.github.io/sprint-content/#contributors), alongside AGU.

Reviewer: ```suggestion
For many, 2023 is the [Year of Open Science](https://www.whitehouse.gov/ostp/news-updates/2023/01/11/fact-sheet-biden-harris-administration-announces-new-actions-to-advance-open-and-equitable-research/). OLS and [MetaDocencia (MD)](https://www.metadocencia.org/) are delighted to announce that we will be participating in the Year of Open Science by **delivering Open Science training** for **[NASA](https://www.nasa.gov/)**, using materials originally developed by [a huge group of open researchers worldwide](https://opensciency.github.io/sprint-content/#contributors), alongside [American Geophysical Union (AGU)](https://www.agu.org/).
```

## What will we do? 

> Spanish is the second most spoken language in the United States, after English. When NASA launched the grant call, MD & OLS partnered to propose a dual Spanish/English language set of training cohorts using our expertise in online virtual cohort training and Spanish-language education. [We are delighted to share that we've been awarded a three-year grant to deliver this training!](https://www.nasa.gov/centers/marshall/news/releases/2023/nasa-boosts-open-science-through-innovative-training/)

Reviewer: ```suggestion
Spanish is the **second most spoken language in the United States**, after English. When NASA launched the grant call, MD & OLS partnered to propose a **dual Spanish/English language set of training cohorts** using our expertise in online virtual cohort training and Spanish-language education. [**We are delighted to share that we've been awarded a three-year grant to deliver this training!**](https://www.nasa.gov/centers/marshall/news/releases/2023/nasa-boosts-open-science-through-innovative-training/)
```

The training will cover [the five modules in OpenCore](https://opensciency.github.io/sprint-content), and participants will work with a mentor to discuss real-world application of the skills they use. The schedule for individual cohorts is shown below: 

> ![Gantt chart showing five training modules taking 2.5 hours each, as well as 30 minutes of mentor discussion, and an hour of self-reflection each week, totalling 4 hours per week for six weeks. The final week is a live-streamed graduation presentation.](https://hackmd.io/_uploads/Hk-H2tEb2.png)

Reviewer: It would be more accessible to have this image as a table in the text directly. Is it possible?

Reviewer 2: @NPDebs can you make it so? 🙏  let me know if having access to the table in google sheets would help

Author: Yes, please. That would help a lot, @yochannah. Thanks!

Reviewer 2: sent to you!

> We plan to run five cohorts per year. 

Reviewer: ```suggestion
We plan to run **five cohorts per year**. 
```

Reviewer: This sentence looks a bit lonely below

  
## What's the difference between this and OLS's traditional mentorship training? 

> We foresee this being very similar to what we already did, but more condensed - rather than ~2 hours over 16 weeks, participants will be spending ~4 hours over 6 weeks. We may also add additional seminars and group discussions to cover topics that were in the OLS curriculum historically, but that aren't part of OpenCore. 

Reviewer: ```suggestion
We foresee this being very similar to what we already did, but **more condensed** - rather than ~2 hours over 16 weeks, participants will be spending ~4 hours over 6 weeks. We may also add **additional seminars and group discussions** to cover topics that were in the OLS curriculum historically, but that aren't part of OpenCore. 
```

## What does this mean for OLS and MetaDocencia? 

> In a word: Sustainability. We're delighted to have a three-year roadmap to help us keep doing the work we love. We'd also spent time brainstorming ways to collaborate as organisations, and one of our biggest joint dreams was to run "EspañOLS" - Spanish-language-first open research training.

Reviewer: ```suggestion
In a word: **Sustainability**. 

We're delighted to have a **three-year roadmap** to help us keep **doing the work we love**. We'd also spent time **brainstorming ways to collaborate** as organisations, and one of our biggest **joint dreams** was to run *EspañOLS* - Spanish-language-first open research training.
```

> After OLS-8, we'll probably take a break from the longer-format mentoring cohorts (recently rebranded to be "Open Seeds"), at the very least until we get comfortable with running these shorter, more intense cohorts. This break will also give us time to [further evaluate and understand the impact of OLS-1 to -8](https://openlifesci.org/posts/2022/02/18/ols-is-hiring/), and to invest into [localizing Open Seeds](https://openlifesci.org/posts/2022/12/20/ECB-grant-announcement/). That said - one of the biggest focuses for OLS and MD has _always_ been global community, so we're still looking for different ways to fund and support our networking, our global opportunity and skillsharing, and the compassion and love we get from you all. 

Reviewer: ```suggestion
After OLS-8, we'll **probably take a break from the longer-format mentoring cohorts** (recently rebranded to be [*Open Seeds*]({% link  _posts/2023-04-18-Introducing-the-new-OLS.md %}), at the very least until we get comfortable with running these shorter, more intense cohorts. This break will also give us time to [further **evaluate and understand the impact** of OLS-1 to OLS-8]({% link  posts/2022-02-18-ols-is-hiring.md %}), and to invest into [**localizing Open Seeds**]({% link _posts/2022-12-20-ECB-grant-announcement.md %}). 

That said - one of the biggest focuses for OLS and MD has **always** been **global community**, so we're still **looking for different ways** to **fund and support** our **networking**, our **global opportunity** and **skillsharing**, and the compassion and love we get from you all. 
```

> 💌 Do you have ideas you want to see, fund, commision us to deliver? Ping team@openlifesci.org, or message us on Slack. 

Reviewer: ```suggestion
<i class="fa-solid fa-envelope-open-text"></i> **Do you have ideas you want to see, fund, commision us to deliver?** Ping [{{ site.email|replace:'@','[at]' }}](mailto:{{ site.email }}), or message us on Slack. 
```

---
layout: post
title: "One small step for NASA, one big step for open research"

> subtitle: "OLS and MetaDocencia to deliver OpenCore Science Training"

Reviewer: ![image](https://github.com/open-life-science/open-life-science.github.io/assets/1842467/14f81275-f096-4904-9857-e9e0454ba48c)

The subtitle is not displayed on the top. Maybe we can use `description` instead (as in the `index.md`)

Author: Yes, I think that might be should...

Author: Yes, I think that might be better...

authors:
- yochannah
image: https://images.unsplash.com/photo-1502101872923-d48509bff386?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1032&q=80
photos:

>   name: Photo by Jukan Tateisi on Unsplash

Reviewer: ```suggestion
  name: Jukan Tateisi
```

>   url: https://unsplash.com/@tateisimikito

Reviewer: It should be the link to the image, not the author

Author: Done. Thank you!

> ---

Reviewer: Can you also add a license? Thanks

Author: Photos under _unsplash_ are  CC zero. I put there `CC-BY`. Is that okay?

Author: Okay. Thanks!
